test/api/run

```diff
@@ -99,6 +99,15 @@ query_api() {
     echo "$out"
 }
 
+# Test that scyla is serving the api port.
```

Comment:
This comment is wrong (you copied it from somewhere?)

Reply:
It was adapted from the redis test, and should have been moved indeed down to the loop that's probing the api.